fastlane/lib/fastlane/auto_complete.rb

```diff
@@ -0,0 +1,16 @@
+module Fastlane
+  # Enable tab auto completion
+  class AutoComplete
+    def self.execute
+      path = File.join(Fastlane::Helper.gem_path('fastlane'), 'lib', 'assets', 'completions')
+
+      fastlane_conf_dir = "~/.fastlane"
+      `mkdir -p #{fastlane_conf_dir} && cp -r #{path} #{fastlane_conf_dir}`
+
+      UI.success "To use auto complete for fastlane"
+      UI.success "add the following line to your favorite rc file (e.g. ~/.bashrc)"
```

Comment:
The line wrap feels a little awkward to me, here. Any reason not to just say it all in one statement with a comma?